test/exptest.c

```diff
@@ -39,10 +32,9 @@ static int a_is_zero_mod_one(const char *method, const BIGNUM *r,
                              const BIGNUM *a)
 {
     if (!BN_is_zero(r)) {
-        fprintf(stderr, "%s failed:\n", method);
-        fprintf(stderr, "a ** 0 mod 1 = r (should be 0)\n");
-        BN_print_var(a);
-        BN_print_var(r);
+        TEST_error("%s failed: a ** 0 mod 1 = r (should be 0)", method);
+        test_output_bignum("a", a);
+        test_output_bignum("r", r);
```

Comment:
Only because I replaced the whole function before fixing the macro.
I'll switch it back to the macro.